x-pack/plugins/ml/common/types/anomaly_detection_jobs/datafeed.ts

```diff
@@ -15,4 +15,9 @@ export type ChunkingConfig = estypes.ChunkingConfig;
 
 export type Aggregation = Record<string, estypes.AggregationContainer>;
 
-export type IndicesOptions = estypes.IndicesOptions;
+// export type IndicesOptions = estypes.IndicesOptions;
```

Comment:
Is this commented code needed?

Reply:
I left the commented version in so we can switch back to using it once the esclient types are correct.
But it can be removed.